ExampleApp/ViewController.swift

```diff
@@ -164,6 +176,10 @@ class ViewController: UITableViewController {
             // Number of items in a row (supported values > 0)
             imagePicker.layoutConfiguration.numberOfAssetItemsInRow = assetItemsInRow
             
+            // Enable assets drag & drop
+            imagePicker.layoutConfiguration.enableAssetDragAndDrop = dragAndDropConfig
```

Comment:
this should also be available only for ios 11 +

Reply:
Its example app, on <ios11 its ignored